vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -55,6 +55,22 @@
       <Configuration>Release</Configuration>
       <Platform>x64</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="WinUI3|ARM">
```

Comment:
>WinUI3 [](start = 35, length = 6)

Having configurations like this is very intrusive. What if tomorrow I want to introduce another flag or two flags. Should we grow the matrix of all possible configurations?

Reply:
I do not see a better way to achieve the same yet. If we use configurations, then should we have them for debug and release? E.g. WinUI3_Debug and WinUI3_Release?

Reply:
we don't need that - the whole point of this new configuration is that it's temporary until we make WinUI3 the default which should be soonish, and that we don't let the code become non-WinUI3 friendly (i.e. don't use WUX types, etc.).
Once we are able to, we'll delete the WinUI3 configuration and make Debug/Release always target it winui 3.